diff --git a/pkg/redfishapi/model_resource_status_conditions_inner.go b/pkg/redfishapi/model_resource_status_conditions_inner.go
--- a/pkg/redfishapi/model_resource_status_conditions_inner.go
+++ b/pkg/redfishapi/model_resource_status_conditions_inner.go
@@ -8,6 +8,8 @@
  */
 
 package redfishapi
+
+import "time"
 
 type ResourceStatusConditionsInner struct {
 	ConditionType *ResourceV1220ConditionType `json:"ConditionType,omitempty"`
